components/brave_rewards/extension/brave_rewards/manifest.json

```diff
@@ -0,0 +1,35 @@
+{
+  "version": "1.0.0",
+  "name": "Brave Rewards",
+  "manifest_version": 2,
+  "description": "Brave Rewards Extension",
+  "default_locale": "en",
+  "background" : {
+    "scripts": ["out/brave_rewards_panel_background.bundle.js"],
+    "persistent": true
+  },
+  "permissions": [
+    "tabs",
+    "chrome://favicon/*"
+  ],
+  "browser_action": {
+    "default_popup": "brave_rewards_panel.html",
+    "default_icon": {
+      "16": "img/bat-16.png",
+      "32": "img/bat-32.png",
+      "48": "img/bat-48.png",
+      "64": "img/bat-64.png",
+      "128": "img/bat-128.png",
+      "256": "img/bat-256.png"
+    }
+  },
+  "icons": {
+    "16": "img/bat-16.png",
+    "32": "img/bat-32.png",
+    "48": "img/bat-48.png",
+    "64": "img/bat-64.png",
+    "128": "img/bat-128.png",
+    "256": "img/bat-256.png"
+  },
```

Comment:
@NejcZdovc I think we should add csp for this extension.
You could check brave-extension and webtorrent extension's manifest for reference and only relax the policy for things that are necessary.

Reply:
btw this is the default CSP when one is not specified: `script-src 'self'; object-src 'self'` according to https://developer.chrome.com/extensions/contentSecurityPolicy

this is already pretty restrictive. if we wanted to restrict it more, i think `script-src 'self'` would probably be sufficient

Reply:
@yrliou CSP added